genon/preprocessor/facade/gitbook_doc/guardrail_workflow_setup.md

```
@@ -0,0 +1,358 @@
+워크플로우** 에 위임해 판별하고, 그 결과를 청크 메타(`content_category`) 라벨과 (옵션) 마스킹 치환으로
+반영하는 기능의 통합 가이드입니다.
+
+- **1~2절**: 전처리기를 쓰는 쪽(호출 방법, on/off 스위치, config 설정)
+- **3~4절**: 워크플로우를 준비하는 쪽(GenOS — 입출력 형태, 배포)
+- 카테고리 정의·프롬프트·정규식 필터는 모두 **GenOS의 워크플로우에서 설정**되며, 전처리기는 워크플로우 응답을 반영만 합니다.
+
+---
+
+## 0. 전체 그림
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)